src/components/widget_types/twitter.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <div>
+        <v-card v-for="tweet in tweets.slice(0, numberTweets)" v-bind:key="tweet.text">
+            <v-card-title>
+                <v-icon
+                    left
+                >
+                    mdi-twitter
+                </v-icon>
+                <span class="title font-weight-light">Twitter</span>
+            </v-card-title>
```

Comment:
I'm wondering if the title up here should be the userID that you have below. I think usually the top of a tweet is like their profile picture and name and their handle?